backend/tests/core/Entities/BuildingHelper.cs

```diff
@@ -25,8 +39,9 @@ public static partial class EntityHelper
         public static Entity.Building CreateBuilding(Entity.Parcel parcel, int id, string projectNumber = null, string name = null, int lat = 0, int lng = 0, Entity.Agency agency = null)
         {
             projectNumber ??= $"p{id}";
-            agency ??= parcel.Agency;
-            var address = EntityHelper.CreateAddress(++parcel.AddressId, parcel.Address.Address1, parcel.Address.Address2, parcel.Address.AdministrativeArea, parcel.Address.Province, parcel.Address.Postal);
+            agency ??= parcel?.Agency ?? EntityHelper.CreateAgency(id);
```

Comment:
stop depending on the parcel being populated, if it isn't, generate a value.

Reply:
what's the ??= operator?

Reply:
this is the same as agency = agency != undefined ? agency : **something else**